snuba/admin/views.py

```diff
@@ -111,6 +113,54 @@ def migrations_groups_list(group: str) -> Response:
     return make_response(jsonify({"error": "Invalid group"}), 400)
 
 
+class MigrationActionConverter(BaseConverter):
+    regex = r"(?:run|reverse)"
+
+
+application.url_map.converters["migration_action"] = MigrationActionConverter
+
+
+@application.route(
+    "/migrations/<group>/<migration_action:action>/<migration_id>",
+    methods=["POST"],
+)
+def run_or_reverse_migration(group: str, action: str, migration_id: str) -> Response:
+    if group not in settings.ADMIN_ALLOWED_MIGRATION_GROUPS:
+        return make_response(jsonify({"error": "Group not allowed"}), 400)
+
+    runner = Runner()
+    migration_group = MigrationGroup(group)
+    migration_key = MigrationKey(migration_group, migration_id)
```

Comment:
How are we going to filter out and reject dangerous migrations ?

Reply:
The plan is to add some middleware that will check for safety. That will be in a separate PR.  This one just simply checks if the group is allowed.